DOCSUP-4266: Document the SQL-statement ALTER UPDATE/DELETE ... IN PARTITION

sevirov opened the pull request:
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #13403

sevirov committed:
Update partition.md

Добавил новый подраздел UPDATE | DELETE IN PARTITION.

sevirov committed:
Update partition.md

Правлю битую ссылку.

sevirov committed:
Merge remote-tracking branch 'remotes/clickhouse/master' into sevirov-DOCSUP-4266-document_alter_update_delete_partition

sevirov committed:
Merge remote-tracking branch 'remotes/clickhouse/master' into sevirov-DOCSUP-4266-document_alter_update_delete_partition

sevirov committed:
Update partition.md

Внес поправки в общий вид запроса IN PARTITION.

sevirov committed:
Update partition.md

Внес поправки в английскую версию.

BayoNet commented on docs/en/sql-reference/statements/alter/partition.md:
```diff
@@ -233,6 +233,46 @@ ALTER TABLE hits MOVE PART '20190301_14343_16206_438' TO VOLUME 'slow'
 
+## UPDATE IN PARTITION {#update-in-partition}
+
+Allows to manipulate data in the specifies partition matching the specified filtering expression. Implemented as a [mutation](../../../sql-reference/statements/alter/index.md#mutations).
```
```suggestion
Manipulates data in the specifies partition matching the specified filtering expression. Implemented as a [mutation](../../../sql-reference/statements/alter/index.md#mutations).
```

BayoNet commented on docs/en/sql-reference/statements/alter/partition.md:
```diff
@@ -233,6 +233,46 @@ ALTER TABLE hits MOVE PART '20190301_14343_16206_438' TO VOLUME 'slow'
+
+## DELETE IN PARTITION {#delete-in-partition}
+
+Allows to delete data in the specifies partition matching the specified filtering expression. Implemented as a [mutation](../../../sql-reference/statements/alter/index.md#mutations).
```
```suggestion
Deletes data in the specifies partition matching the specified filtering expression. Implemented as a [mutation](../../../sql-reference/statements/alter/index.md#mutations).
```

BayoNet commented on docs/en/sql-reference/statements/alter/partition.md:
````diff
@@ -250,4 +290,6 @@ All the rules above are also true for the [OPTIMIZE](../../../sql-reference/stat
 OPTIMIZE TABLE table_not_partitioned PARTITION tuple() FINAL;
 ```
 
+`IN PARTITION` specifies the partition to which the [UPDATE](../../../sql-reference/statements/alter/update.md#alter-table-update-statements) or [DELETE](../../../sql-reference/statements/alter/delete.md#alter-mutations) expressions will be applied as a result of the query `ALTER TABLE`. New parts will be created only from the specified partition. In this way, `IN PARTITION` helps reduce the load ...
````
```suggestion
`IN PARTITION` specifies the partition to which the [UPDATE](../../../sql-reference/statements/alter/update.md#alter-table-update-statements) or [DELETE](../../../sql-reference/statements/alter/delete.md#alter-mutations) expressions are applied as a result of the `ALTER TABLE` query. New parts are created only from the specified partition. In this way, `IN PARTITION` helps to reduce the load when the table is divided into many partitions, and you only need to update the data point-by-point.
```

sevirov committed:
Update files in the 'alter'.

Внес поправки в английскую версию, выполнил перевод на русский язык, подкорректировал некоторые файлы в 'ALTER'.
